subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt_priv.h

```diff
@@ -4,59 +4,138 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/**
+ * @file
+ * @brief Declares implementation-specific functions required by image management.  The default
+ * stubs can be overridden with functions that are compatible with the host OS.
```

Comment:
These aren't weak functions and can't be overridden

Reply:
They never were. I should fix the doc.

Reply:
Comment about weak stubs can go